js/router.js

```diff
@@ -38,6 +38,11 @@ define(['jquery', 'underscore', 'backbone', 'views/home/home', 'views/home/calen
 
   var initialize = function() {
       router = new AppRouter;
+
+      // Maybe this is the wrong place to call this, but 
+      // the hamburger menu applies everywhere.
+      hamburgerMenuView = new hamburgerMenuView();
```

Comment:
I changed the constructor to pass in the el . All of the other views hardcode their el variables, though. I did not change those.

I am really struggling with moving the constructor for the `hamburgerMenuView`. I am pretty sure it does not belong in `views/home/home.js` by itself, because if you navigate to watcamp.com/#links directly then the hamburger menu is broken again. `home.js` is not a base view; it is specifically for the front page. 

I agree that invoking `hamburgerMenuView()` in the constructor of `router.js` seems wrong. I just do not understand what is right. As far as I can tell `router.js` is the only place where views are initialized.